Sphinx indicates failure with -v but not without it

```
(karellen-venv) macbookpro-1ce1:karellen-stack-core arcivanov$ pyb sphinx_generate_documentation
PyBuilder version 0.11.9.dev20160324115053
Build started at 2016-04-13 16:33:36
------------------------------------------------------------
[INFO]  Downloading or updating plugin pypi:karellen_pyb_plugin version >=0.0.1.dev
[INFO]  Installed or updated plugin pypi:karellen_pyb_plugin version >=0.0.1.dev.
[INFO]  Building karellen-stack-core version 0.0.1.dev (0.0.1.dev20160413203337)
[INFO]  Executing build in /Users/arcivanov/Documents/src/karellen/karellen-stack-core
[INFO]  Going to execute task sphinx_generate_documentation
[INFO]  Running sphinx_generate_documentation
------------------------------------------------------------
BUILD SUCCESSFUL
------------------------------------------------------------
Build finished at 2016-04-13 16:33:38
Build took 2 seconds (2190 ms)
```

vs

```
(karellen-venv) macbookpro-1ce1:karellen-stack-core arcivanov$ pyb sphinx_generate_documentation -v
PyBuilder version 0.11.9.dev20160324115053
Build started at 2016-04-13 16:33:42
------------------------------------------------------------
[INFO]  Downloading or updating plugin pypi:karellen_pyb_plugin version >=0.0.1.dev
[INFO]  Installed or updated plugin pypi:karellen_pyb_plugin version >=0.0.1.dev.
[INFO]  Building karellen-stack-core version 0.0.1.dev (0.0.1.dev20160413203342)
[INFO]  Executing build in /Users/arcivanov/Documents/src/karellen/karellen-stack-core
[INFO]  Going to execute task sphinx_generate_documentation
[INFO]  Running sphinx_generate_documentation
[INFO]  sphinx-build -b html /Users/arcivanov/Documents/src/karellen/karellen-stack-core/docs /Users/arcivanov/Documents/src/karellen/karellen-stack-core/docs/_build/
------------------------------------------------------------
BUILD FAILED - Sphinx build command failed. See /Users/arcivanov/Documents/src/karellen/karellen-stack-core/target/reports/sphinx_generate_documentation for details.
------------------------------------------------------------
Build finished at 2016-04-13 16:33:44
```

--- a/src/main/python/pybuilder/plugins/python/sphinx_plugin.py
+++ b/src/main/python/pybuilder/plugins/python/sphinx_plugin.py
@@ -89,11 +89,13 @@ def run_sphinx_build(build_command, task_name, logger, project):
     logger.info("Running %s" % task_name)
     log_file = project.expand_path(
         "$dir_target/reports/{0}".format(task_name))
+
     if project.get_property("verbose"):
         logger.info(build_command)
-        exit_code = execute_command(build_command, log_file, shell=True)
-        if exit_code != 0:
-            raise BuildFailedException("Sphinx build command failed. See %s for details.", log_file)
+
+    exit_code = execute_command(build_command, log_file, shell=True)
+    if exit_code != 0:
+        raise BuildFailedException("Sphinx build command failed. See %s for details.", log_file)
 
 
 @task("sphinx_generate_documentation", "Generates documentation with sphinx")

--- a/src/unittest/python/plugins/python/sphinx_plugin_tests.py
+++ b/src/unittest/python/plugins/python/sphinx_plugin_tests.py
@@ -16,9 +16,10 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-from unittest import TestCase
-from mock import Mock, patch
 from logging import Logger
+from unittest import TestCase
+
+from mock import Mock, patch
 
 from pybuilder.core import Project, Author
 from pybuilder.plugins.python.sphinx_plugin import (
@@ -26,14 +27,13 @@ from pybuilder.plugins.python.sphinx_plugin import (
     assert_sphinx_quickstart_is_available,
     get_sphinx_build_command,
     get_sphinx_quickstart_command,
-    initialize_sphinx_plugin)
+    initialize_sphinx_plugin,
+    run_sphinx_build)
 
 
 class CheckSphinxAvailableTests(TestCase):
-
     @patch('pybuilder.plugins.python.sphinx_plugin.assert_can_execute')
     def test_should_check_that_sphinx_can_be_executed(self, mock_assert_can_execute):
-
         mock_logger = Mock(Logger)
 
         assert_sphinx_is_available(mock_logger)
@@ -44,7 +44,6 @@ class CheckSphinxAvailableTests(TestCase):
 
     @patch('pybuilder.plugins.python.sphinx_plugin.assert_can_execute')
     def test_should_check_that_sphinx_quickstart_can_be_executed(self, mock_assert_can_execute):
-
         mock_logger = Mock(Logger)
 
         assert_sphinx_quickstart_is_available(mock_logger)
@@ -54,7 +53,6 @@ class CheckSphinxAvailableTests(TestCase):
 
 
 class SphinxPluginInitializationTests(TestCase):
-
     def setUp(self):
         self.project = Project("basedir")
 
@@ -107,12 +105,10 @@ class SphinxPluginInitializationTests(TestCase):
 
 
 class SphinxBuildCommandTests(TestCase):
-
     def setUp(self):
         self.project = Project("basedir")
 
     def test_should_generate_sphinx_build_command_per_project_properties(self):
-
         self.project.set_property("sphinx_config_path", "docs/")
         self.project.set_property("sphinx_source_dir", "docs/")
         self.project.set_property("sphinx_output_dir", "docs/_build/")
@@ -124,7 +120,6 @@ class SphinxBuildCommandTests(TestCase):
                          "sphinx-build -b JSONx basedir/docs/ basedir/docs/_build/")
 
     def test_should_generate_sphinx_quickstart_command_with_project_properties(self):
-
         self.project.set_property("sphinx_doc_author", "bar")
         self.project.set_property("sphinx_project_name", "foo")
         self.project.set_property("sphinx_project_version", "3")
@@ -134,3 +129,12 @@ class SphinxBuildCommandTests(TestCase):
 
         self.assertEqual(sphinx_quickstart_command,
                          "sphinx-quickstart -q -p 'foo' -a 'bar' -v 3 basedir/docs/")
+
+    @patch('pybuilder.plugins.python.sphinx_plugin.execute_command', return_value=0)
+    def test_should_execute_command_regardless_of_verbose(self, exec_command):
+        self.project.set_property("verbose", True)
+        self.project.set_property("dir_target", "spam")
+        initialize_sphinx_plugin(self.project)
+
+        run_sphinx_build("foo", "bar", Mock(), self.project)
+        self.assertEquals(exec_command.call_count, 1)

